packages/node_modules/pouchdb-core/src/setup.js

```diff
@@ -122,4 +123,8 @@ PouchDB.defaults = function (defaultOpts) {
   return PouchAlt;
 };
 
+PouchDB.fetch = function (url, opts) {
```

Comment:
Could this just be ``PouchDB.fetch = fetch``? Both the browser and ``fetch-cookie`` at least currently don't export more than a plain function.

Reply:
Yeh changed that cheers